### PR TITLE
tidb-in-kubernetes: fix pv deletion

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

Update the PV deletion method.

### What is the related PR or file link(s)? 

- [ ] Reference link(s):
- [x] This PR is to align with: https://github.com/pingcap/docs-cn/pull/2155
- [ ] N/A (not applicable)

### Which TiDB version(s) does your changes apply to? 

- [ ] All active versions: dev, v3.0, v2.1, v3.1
- [x] dev (the latest development version)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] All active and inactive versions
- [ ] N/A (not applicable)

**Note:** If your changes apply to multiple TiDB versions, make sure you update the documents in the **corresponding** version folders such as "dev", "v3.0", "v2.1" and "v3.1" in this PR.

- [ ] Updated one version first. Will update other versions after I get two LGTMs.

### CODE REVIEW

**Comment by cofyc** [comment]
LGTM

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 1]
```diff
@@ -260,12 +260,22 @@ In general, after a PVC is no longer used and deleted, the PV bound to it is rec
 >
 > By default, to ensure data safety, TiDB Operator automatically changes the reclaim policy of the PVs of PD and TiKV to `Retain`.
 
-When the reclaim policy of PVs is set to `Retain`, if the data of a PV can be deleted, you need to set the reclaim policy of the PV to `Delete`. In this case, as long as the corresponding PVC is deleted, the PV is automatically deleted and reclaimed.
+When the reclaim policy of PVs is set to `Retain`, if the data of a PV can be deleted, delete PV and the corresponding data according to the following steps:
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
When the reclaim policy of PVs is set to `Retain`, if the data of a PV can be deleted, delete this PV and the corresponding data according to the following steps:
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 2]
```diff
@@ -260,12 +260,22 @@ In general, after a PVC is no longer used and deleted, the PV bound to it is rec
...
-```shell
-kubectl patch pv <pv-name> -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
-```
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl delete pvc <pvc-name> --namespace=<namespace>
+    ```
+
+2. Set the reclaim policy of the PV to `Delete`, then the PV is automatically deleted and reclaimed.
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
2. Set the reclaim policy of the PV to `Delete`. Then the PV is automatically deleted and reclaimed.
```

**Comment by TomShawn** [comment]
@CaitinChen Comments addressed. PTAL again, thanks!

**Pull request merged**